setup.cfg

```diff
@@ -15,6 +15,8 @@ ignore =
     W503
     E265
     F811
+    # Allows type hinting as Gridded[DataArray, "(X:center)"], where we did `from typing import Annotated as Gridded`
+    F722
```

Comment:
So I wanted to get type hinting to look like this
```python
@as_grid_ufunc
def f(a: Gridded[xr.DataArray, "X:center"]):
   ...
```
instead of this
```python
@as_grid_ufunc
def f(a: Annotated[xr.DataArray, "X:center"]):
   ...
```
but after much messing about I think the only way to achieve that whilst retaining `mypy` compatibility is the hacky trick of doing
```python
from typing import Annotated as Gridded
```
As far as I can tell this is perfectly safe, but it does then mean I need to silence this flake8 warning.

I think using `Gridded` is clearer for users, but maybe others think this is unnecessary?